kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/Identity.kt

```diff
@@ -0,0 +1,10 @@
+package com.clouway.kcqrs.core
+
+import java.time.Instant
+
+/**
+ * Identity is representing a single identity which represents the author of the request.
+ *
+ * @author Miroslav Genov (miroslav.genov@clouway.com)
+ */
+data class Identity(val id: String, val time: Instant)
```

Comment:
In the description of the class is described that Object represents the author. I think a better name will be ``` AuthorIdentity  ``` if it`s not used for another purpose.

Reply:
Replaced author with initiator.